src/Cms/Core.php

```diff
@@ -331,33 +331,34 @@ public function load(): Loader
-			'site'        => fn (array $roots) => $roots['index'] . '/site',
-			'accounts'    => fn (array $roots) => $roots['site'] . '/accounts',
-			'blueprints'  => fn (array $roots) => $roots['site'] . '/blueprints',
-			'cache'       => fn (array $roots) => $roots['site'] . '/cache',
-			'collections' => fn (array $roots) => $roots['site'] . '/collections',
-			'commands'    => fn (array $roots) => $roots['site'] . '/commands',
-			'config'      => fn (array $roots) => $roots['site'] . '/config',
-			'controllers' => fn (array $roots) => $roots['site'] . '/controllers',
-			'languages'   => fn (array $roots) => $roots['site'] . '/languages',
-			'license'     => fn (array $roots) => $roots['config'] . '/.license',
-			'logs'        => fn (array $roots) => $roots['site'] . '/logs',
-			'models'      => fn (array $roots) => $roots['site'] . '/models',
-			'plugins'     => fn (array $roots) => $roots['site'] . '/plugins',
-			'sessions'    => fn (array $roots) => $roots['site'] . '/sessions',
-			'snippets'    => fn (array $roots) => $roots['site'] . '/snippets',
-			'templates'   => fn (array $roots) => $roots['site'] . '/templates',
-			'roles'       => fn (array $roots) => $roots['blueprints'] . '/users',
+			'i18n:rules'   => fn (array $roots) => $roots['i18n'] . '/rules',
+
+			'index'        => fn (array $roots) => static::$indexRoot ?? dirname(__DIR__, 3),
+			'assets'       => fn (array $roots) => $roots['index'] . '/assets',
+			'content'      => fn (array $roots) => $roots['index'] . '/content',
+			'media'        => fn (array $roots) => $roots['index'] . '/media',
+			'panel'        => fn (array $roots) => $roots['kirby'] . '/panel',
+			'site'         => fn (array $roots) => $roots['index'] . '/site',
+			'accounts'     => fn (array $roots) => $roots['site'] . '/accounts',
+			'blueprints'   => fn (array $roots) => $roots['site'] . '/blueprints',
+			'cache'        => fn (array $roots) => $roots['site'] . '/cache',
+			'collections'  => fn (array $roots) => $roots['site'] . '/collections',
+			'commands'     => fn (array $roots) => $roots['site'] . '/commands',
+			'config'       => fn (array $roots) => $roots['site'] . '/config',
+			'controllers'  => fn (array $roots) => $roots['site'] . '/controllers',
+			'languages'    => fn (array $roots) => $roots['site'] . '/languages',
+			'license'      => fn (array $roots) => $roots['config'] . '/.license',
+			'logs'         => fn (array $roots) => $roots['site'] . '/logs',
+			'models'       => fn (array $roots) => $roots['site'] . '/models',
+			'plugins'      => fn (array $roots) => $roots['site'] . '/plugins',
+			'roles'        => fn (array $roots) => $roots['blueprints'] . '/users',
+			'sessions'     => fn (array $roots) => $roots['site'] . '/sessions',
+			'snippets'     => fn (array $roots) => $roots['site'] . '/snippets',
+			'templates'    => fn (array $roots) => $roots['site'] . '/templates',
+			'translations' => null,
```

Comment:
`Question:` I don't have a good idea yet, but wondering if it's too confusing to have `i18n:translations` and `translations`? Maybe `languages:translations`?

Reply:
@bastianallgeier What do you think?

Reply:
What was our last decision for the language variable permissions? I wonder if we maybe should go the same route here with naming as for permissions?

Reply:
For now to be `languageVariables.create|update|delete`.

Reply:
We could go for `language:variables` maybe

I know that it totally clashes with the translations naming everywhere. But I think it makes sense to fully commit to one naming convention here and language variables are at least not giving us any more headaches with the entire content translations vs. ui translations monster.

Reply:
In the long run, what if we also changed the `translations` index in the language file to `variables`? For ex: we can deprecate `translations` index v5-v6 and we can remove v7. How does that sound?

Reply:
Yes, we should definitely do that. Maybe it would be good to already use LanguageVariables for the class name in this PR and maybe even refactor the property name?

Reply:
Done for this PR. I'll deprecate the `translations` index in another PR.

Reply:
Maybe even `i18n:variables` then?

Reply:
Not sure for this PR, @bastianallgeier ?

Reply:
I would say that it belongs to the language and thus the language namespace feels better to me.

Reply:
My point there is that so far we don't have a language namespace anyways, we are creating a new one. And what if language is the i18n namespace?